main.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python2
+
+import argparse
+import sys
+import re
+import requests
+from HTMLParser import HTMLParser
+
+
+def web_request(website):
+    """This is a handler """
+    print(website)
+    r = requests.get(website)
+
+    parser = MyParser()
+    parser.feed(r.content)
+    phone_scrape(parser.data_in_tags)
+    email_scrape(parser.data_in_tags)
+    url_scrape(parser.url_to_scrape)
+
+
+def phone_scrape(website):
```

Comment:
This function doesn't seem to be producing any phone numbers. Check out how your scraper is actually grabbing the content, I don't think it is all there.

Reply:
This is all good.